### ISSUE TITLE
Incorrect method names for JS HubConnectionBuilder 

### ISSUE THREAD

**Issue by cbook-lam** [dotnet/AspNetCore.Docs#32284]
### Description

For the JS HubConnectionBuilder, `withServerTimeoutInMilliseconds` and `withKeepAliveIntervalInMilliseconds` are listed as functions, but those names do not exist. They are `withServerTimeout` and `withKeepAliveInterval` respectively. The corrected sample is:

    var connection = new signalR.HubConnectionBuilder()
	    .withUrl("/chatHub")
	    .withServerTimeout(60000)
	    .withKeepAliveInterval(30000)
	    .build();

The table "Additional options for configuring timeout and keep-alive behavior" also needs to be corrected.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/signalr/configuration?view=aspnetcore-8.0&tabs=javascript#configure-timeout-and-keep-alive-options

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/signalr/configuration.md

### Document ID

66d252c3-6300-7e28-9aa0-ef2ff66e7a76

### Article author

@bradygaster

**Comment by guardrex** [comment]
@wadepickett ... I'm going have an 👁️ & 👂open for the updates here. I'm currently trying to get some related Blazor content updated for the second time due to API churn over the years on https://github.com/dotnet/AspNetCore.Docs/pull/32237.

**Comment by wadepickett** [comment]
@Guardrex, Thanks for the heads up on the related content issue. I'm not going to be able to address this doc issue during this sprint that is already underway that I am racing through.  If you need any immediate related guidance questions to keep from being blocked, you might have to check in with Brennan.

**Comment by wadepickett** [comment]
@guardrex, A quick look at a package for [browser/signalr.js for 8.0.0](https://unpkg.com/browse/@microsoft/signalr@8.0.0/dist/browser/signalr.js), (see around line 3456), shows the suggested example to be correct:
```
var connection = new signalR.HubConnectionBuilder()
    .withUrl("/chatHub")
    .withServerTimeout(60000)
    .withKeepAliveInterval(30000)
    .build();
```
I am not sure why it was originally written as `withServerTimeoutInMilliseconds` and `withKeepAliveIntervalInMilliseconds`.  I am guessing those were the original to the early pre-release versions and then they changed just before release.

**Comment by wadepickett** [comment]
Thank you @cbook-lam, we will get it corrected!

**Comment by guardrex** [comment]
I think `withServerTimeoutInMilliseconds` and `withKeepAliveIntervalInMilliseconds` are going to turn out to be <8.0 API.

**Comment by wadepickett** [comment]
@guardrex, yes you are right.  That particular API ref did not get updated, nor did the config doc to the change that removed "milliseconds".

I have expanded the scope of this issue to look through all the 8.0+ docs to update those two old function names.

**Comment by wadepickett** [comment]
Verified there are only two 8.0 docs that contain the pre 8.0 function names:

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/signalr/configuration.md
https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/release-notes/aspnetcore-8.0.md

**Comment by guardrex** [comment]
@MmMapIoSpace ... Please don't double-post. I'll respond to your other comment on the other issue.
